fe/fe-core/src/main/java/com/starrocks/common/MarkedCountDownLatch.java

```diff
@@ -64,6 +64,17 @@ public synchronized boolean markedCountDown(K key, V value) {
         return false;
     }
 
+    public synchronized boolean markedCountDown(K key, V value, Status status) {
+        if (st.ok()) {
+            st = status;
+        }
+        if (marks.remove(key, value)) {
```

Comment:
you can reuse `markedCountDown(key, value)` here.

Reply:
> you can reuse `markedCountDown(key, value)` here.

done, changed in https://github.com/StarRocks/starrocks/pull/23320/commits/1836a3a138a9ea19ff8795e0315bf04e1373a2d1